src/app/components/Search/Instances.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Instances extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.instances = props;
+  }
+
+  parseInstances(instances) {
+    const items = instances.instances.map((instance, key) => (
+      {
+        ebooks: (instance.items) ? instance.items : [],
+        pub_date: (instance.pub_date) ? parseInt(instance.pub_date) : null,
+        pub_place: (instance.pub_place) ? instance.pub_place : null,
+        publisher: (instance.publisher) ? instance.pub_publisher : null,
+        language: (instance.language) ? instance.language : null
+      }
+    ));
+
+    return items;
+  };
+
+  render() {
+    const items = this.parseInstances(this.instances);
+
+    return (
+      <ul className="nypl-instances-list">
+        {
+          items.map((element, key) => (
+            <li className="nypl-results-item">
+              <div className="nypl-results-description">
+                <p tabIndex={key.toString()} key={key.toString()}>
```

Comment:
You don't and shouldn't define `tabIndex` like this. Typically we try not to give element tab indices unless if it's "0" or "-1".

Reply:
`tabIndex` is being removed